client-dotnet/src/EvoMaster.Instrumentation/Coverage/MethodReplacement/DistanceHelper.cs

```diff
@@ -53,20 +82,26 @@ public class DistanceHelper {
         return dist;
     }
 
-    public static int DistanceToChar(char c, char target) {
+    public static int DistanceToChar(char c, char target){
         return Math.Abs(c - target);
     }
 
 
-    public static long GetLeftAlignmentDistance(string a, string b) {
+    public static long GetLeftAlignmentDistance(string a, string b){
         long diff = Math.Abs(a.Length - b.Length);
         var dist = diff * MAX_CHAR_DISTANCE;
 
-        for (var i = 0; i < Math.Min(a.Length, b.Length); i++) {
+        for (var i = 0; i < Math.Min(a.Length, b.Length); i++){
             dist += Math.Abs(a[i] - b[i]);
         }
 
-        Trace.Assert(dist >= 0);
+        if (dist < 0){
+            dist = long.MaxValue; // overflow
+        }
+
+        // Man: Amid, I commented this assert out.
+        // please check why you put such assert here
+        //Trace.Assert(dist >= 0);
         return dist;
```

Comment:
could you check this comment? if you agree with it, then you could remove the comment

Reply:
removed